### PR TITLE
Clean php and phtml files from outcommented Code

### PR DESCRIPTION
**Pull request by shinobu**
This pull request removes most/all outcommented Code in the php and phtml files in OntoWiki (including extensions).

### CODE REVIEW

File: application/views/templates/application/openid.phtml [hunk 1]
```diff
@@ -19,7 +18,6 @@ $(document).ready(function() {
     });
 });
 
-//]]>
```
Comment (white-gecko) [inline application/views/templates/application/openid.phtml]: These should stay

File: application/views/templates/application/webid.phtml [hunk 2]
```diff
@@ -19,7 +18,6 @@ $(document).ready(function() {
     });
 });
 
-//]]>
```
Comment (white-gecko) [inline application/views/templates/application/webid.phtml]: same here

**Review by white-gecko — CHANGES_REQUESTED** [review]
Looks good, just two changes.

**Comment by shinobu** [comment]
done @white-gecko

**Comment by shinobu** [comment]
Tests:
Unit:
OK (86 tests, 154 assertions)
Virtuoso;
Tests: 27, Assertions: 138, Errors: 1. (same as develop branch)
MySQL:
OK (27 tests, 139 assertions)

File: extensions/semanticsitemap/templates/semanticsitemap/sitemap.phtml [hunk 3]
```diff
@@ -11,7 +11,6 @@
 // to avoid problems with these nasty short open tags ...
 echo '<?xml version="1.0" encoding="UTF-8"?>'. PHP_EOL;
 ?>
-<!-- Semantic Sitemap generated by <?php echo $this->appname ?> <?php echo $this->version ?> -->
```
Comment (white-gecko) [inline extensions/semanticsitemap/templates/semanticsitemap/sitemap.phtml]: This should stay as well

File: extensions/navigation/templates/navigation/explore.phtml [hunk 4]
```diff
@@ -5,8 +5,7 @@
  */
 $odd = true;
 ?>
-<!-- <ol class="bullets-none separated">
-    <li> -->
+<!-- <li> -->
```
Comment (white-gecko) [inline extensions/navigation/templates/navigation/explore.phtml]: Why is this still here?

Comment (white-gecko) [reply extensions/navigation/templates/navigation/explore.phtml]: Also the following h2

**Review by white-gecko — CHANGES_REQUESTED** [review]

**Comment by shinobu** [comment]
corrected those things, rebased, squashed

should be OK now @white-gecko

**Review by white-gecko — APPROVED** [review]
LGTM ;-)

**Pull request merged**